docs/Authentication.md

```diff
@@ -936,10 +950,9 @@ import * as React from "react";
 import { Admin } from 'react-admin';
```

Comment:
You need to change the introductory sentence 4 lines above about LogoutButton being passed to `Admin`

Reply:
It's done